### ISSUE TITLE
Setup OpenSearch Dashboard

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#10]
## User story
1. As a user
2. I want a dashboard
3. So that i can interact with the MdHOS from within my browser

## Acceptance criteria
* Setup the OpenSearch Dashboard such that i am able to interact with the OpenSearch instance inside our docker container #8 
* No additional functionality required at this point. Just set up the dashboard that comes with OpenSearch

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
